_events/2021-01-11-foss-backstage.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: event
+title: "FOSS Backstage 2021"
+date: 2022-02-10
+categories: design conference open-source
+eventDate: Wed, 17 - 18 March 2022
+location: Online, Remote and In person in Berlin (limited seats)
+time: 1/2 days
+status: current
+permalink: /2021/01/11/fossback-stage-2021
```

Comment:
also I am guessing the permalink will be `/2021/01/11/fossback-stage-2022` ?

Reply:
I fixed in a new pr i think?